### ISSUE TITLE
yarn start fails: cannot find module 'react-static/babel-preset.js'

### ISSUE THREAD

**Issue by mirkonasato** [nozzle/react-static#913]
### Are you reporting a bug?
Yes

### Environment

1.  `react-static -V`: 6.0.18
2.  `node -v`: v10.14.2
3.  `npm -v`: 6.4.1
4.  Operating system: macOS 10.14.2
5.  Browser and version (if relevant):

### Steps to Reproduce the problem

1. Create a new project:
```
$ react-static create

? What should we name this project? my-blog
? Select a template below... basic
=> Creating new react-static project...
```
2. Start the dev server:
```
$ cd my-blog
$ yarn start
yarn run v1.12.3
$ react-static start
  Error: Cannot find module 'react-static/babel-preset.js' from '/path/to/my-blog'
  [...]
  TypeError: Cannot read property 'DIST' of undefined

  - start.js:85 _callee2$
    [my-blog]/[react-static]/lib/commands/start.js:85:69
```

**Comment by mjsisley** [comment]
Template is installing the wrong version of `react-static`. If you bump the version in `package.json`, it fixes the issue.

**Comment by tannerlinsley** [comment]
Like like we need to fix the template.

**Comment by mirkonasato** [comment]
The `package.json` is actually fine as it says

```
"react-static": "^6.0.2"
```

It's more that the template also comes with a `yarn.lock` file

```
react-static@^6.0.2:
  version "6.0.2"
```

So deleting `yarn.lock` (and `node_modules`) then re-running `yarn` gets the latest version.

Maybe templates shouldn't have a `yarn.lock` file at all?

**Comment by ScriptedAlchemy** [comment]
Yeah, tanner these lock files are great for corporations and company projects. But I’ve had tons of headaches with the my code not working because of lock files  still installing universal version 2. 

 It has definitely gotten better now that we don’t have all those examples, but something to consider

**Comment by tannerlinsley** [comment]
I'm fine not committing the lock files.

**Comment by tannerlinsley** [comment]
Just need a PR to fix the package.json

**Comment by tannerlinsley** [comment]
Fixed in #917
